yapio.c

```diff
@@ -1251,9 +1268,36 @@ yapio_alloc_buffers(const yapio_test_group_t *ytg)
         yapio_destroy_buffers_and_abort();
 }
 
+/* read input md file to initialize yapioMyRank's md array */
+static void
+yapio_initialize_source_md_buffer_from_file(const yapio_test_group_t *ytg)
+{
+    /* each rank reads its own md file '{filename}-md-{rank}' */
+    char md_file[PATH_MAX + 10];
```

Comment:
i hope you never go further than 6 digit numbers (that would be awsome to reach that limit XD)

Reply:
lowering to 4 digit numbers